common/src/chain/transaction/output/mod.rs

```diff
@@ -140,6 +141,8 @@ pub enum TxOutput {
     /// Transfer an output under Hashed TimeLock Contract.
     #[codec(index = 10)]
     Htlc(OutputValue, Box<HashedTimelockContract>),
+    #[codec(index = 11)]
+    AnyoneCanTake(Box<OrderData>),
 }
```

Comment:
Not that I insist on renaming this, but I wonder why this name was chosen. Shouldn't it be "CreateOrder" or something similar?

Reply:
The idea is that the output names describe a use case rather then specific operation performed. Like `Transfer` which doesn't transfer anything per se.